macros.in

```diff
@@ -1322,17 +1322,19 @@ end
     if #arg < 2 then
         macros.error({'not enough arguments'})
     end
+    name = arg[2]
     if arg[1] == 'g' then
         type = 'group'
     elseif arg[1] == 'u' then
         type = 'user'
-    elseif arg[1] == 'r' or arg[1] == 'm' then
+    elseif arg[1] == 'm' and #arg >=3 then
+        type = 'usergroup'
```

Comment:
'groupmember' might be a more obvious thing for what it does. Wouldn't matter a bit in a script internals, but since these are visible in the package dependencies...